perfkitbenchmarker/package_managers.py

```diff
@@ -206,6 +206,7 @@ def InstallPackages(self, packages):
                          '/usr/bin/apt-get -y install %s' % (packages))
       self.RemoteCommand(install_command)
     except errors.VmUtil.SshConnectionError as e:
+      self.RemoteCommand('sudo sed -i "s/azure.//g" /etc/apt/sources.list')
```

Comment:
* Could you expand the pattern matched here? azure<any character> seems overly broad.
* Would you mind adding a TODO to remove this when the package repo is fixed?
* Recommend an argument to `-i`.